docs/PayPalNativeCheckout/README.md

```diff
@@ -0,0 +1,170 @@
+# Accepting PayPal Native Checkout Payments
+
+The PayPal Native Checkout module in the PayPal SDK enables PayPal payments via a native UI experience in your app.
+
+Follow these steps to add PayPal Native Checkout payments:
+
+1. [Setup a PayPal Developer Account](#setup-a-paypal-developer-account)
+1. [Add PayPal Native Checkout Module](#add-paypal-native-checkout-module)
+1. [Test and go live](#test-and-go-live)
+
+## Setup a PayPal Developer Account
+
+You will need to set up authorization to use the PayPal Payments SDK. 
+Follow the steps in [Get Started](https://developer.paypal.com/api/rest/#link-getstarted) to create a client ID. 
+
+You will need a server integration to create an order to capture funds using the [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2). 
+For initial setup, the `curl` commands below can be used as a reference for making server-side RESTful API calls.
+
+## Add PayPal Native Checkout Module
+
+### 1. Add the PayPal Native checkout to your app
+
+#### Swift Package Manager
+
+In Xcode, add the PayPal SDK as a [package dependency](https://developer.apple.com/documentation/swift_packages/adding_package_dependencies_to_your_app) to your Xcode project. Enter https://github.com/paypal/iOS-SDK as the package URL. Tick the "PayPalNativeCheckout" checkbox to add the PayPal Native Checkout package to your app.
+
+In your app's source code files, use the following import syntax to include the PayPal Native Checkout module:
+
+```swift
+import PayPalNativeCheckout
+```
+
+#### CocoaPods
+
+Include the PayPal Native Checkout pod in your `Podfile`.
+
+```ruby
+pod 'PayPalNativeCheckout'
+```
+
+In your app's source files, use the following import syntax to include PayPal's libraries:
+
+```swift
+import PayPalNativeCheckout
+```
+
+### 2. Initiate the Payments SDK
+
+Create a `CoreConfig` using an [access token](../../README.md#access-token):
+
+```swift
+let config = CoreConfig(accessToken: "<ACCESS_TOKEN>", environment: .sandbox)
+```
+
+Create a `PayPalClient` to approve an order with a PayPal payment method:
```

Comment:
I thought [we decided](https://braintree.slack.com/archives/C03U9SEQYHK/p1663773329402289?thread_ts=1663772091.281979&cid=C03U9SEQYHK) to rename this to `PayPalNativeCheckoutClient` to align with web better? Do we have JIRAs to do that work?

Reply:
Yep i missed the thread on slack, going to change it in this PR.